docs/source/basic_tutorial.rst

```diff
@@ -16,6 +16,22 @@ Its goal is to provide researchers:
 - multi-task training over many datasets at once
 - seamless integration of Amazon Mechanical Turk for data collection and human evaluation
 
+Install
+-------
+Follow the step by step guide on how to download and install ParlAI.
+
+1. Clone ParlAI Repository:
+.. code-block:: bash
+        git clone https://github.com/facebookresearch/ParlAI.git
+2. Install Required packages for ParlAI:
+.. code-block:: bash 
+        pip install -r requirements.txt
+3. Install ParlAI on Linux:
+.. code-block:: bash 
+        python setup.py install
+4. Several models has additional requirements:
+        a. DrQA requires installing `PyTorch <http://pytorch.org/>`_
+        b. MemNN requires installing `Lua Torch <http://torch.ch/docs/getting-started.html>`_
```

Comment:
if you click View File, the resulting rst output is still a little off.

I think you need to add more newlines: before and after the code-block lines, in particular.